db/re/item_db_equip.yml

```
@@ -100707,6 +100852,34 @@ Body:
       .@r = getrefine();
       bonus bMaxHPrate,.@r;
       bonus bMaxSPrate,.@r;
+  - Id: 22070
+    AegisName: Applause_Sandal
+    Name: Old Applause Sandals
+    Type: Armor
+    Weight: 200
+    Slots: 1
+    Locations:
+      Shoes: true
+    ArmorLevel: 1
+    EquipLevelMin: 1
+    Refineable: true
+    Script: |
+      bonus bMaxSP,50;
+  - Id: 22071
+    AegisName: Applause_Sandal_
+    Name: Applause Sandal
+    Type: Armor
+    Weight: 200
+    Defense: 7
+    Slots: 1
+    Locations:
+      Shoes: true
+    ArmorLevel: 1
+    EquipLevelMin: 99
+    Refineable: true
+    Script: |
+      bonus bMdef,10;
+      bonus bDelayrate,-2*(getrefine()/2);
```

The description is saying per refine (JRO)
```suggestion
      bonus bDelayrate,-2*getrefine();
```

---

Thanks